src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt

```diff
@@ -88,14 +88,14 @@ object Messages {
         "The one and only %name% has arrived, make way.",
         "%name%, did you bring the pizza, or was that someone else.",
         "Lord %name% has entered.",
-        "I don't know who %name% is but he sounds like a good guy.",
-        "You were the chosen one! It was said you would destroy TPH, not join them! You were my brother, %name%, I loved you!",
+        "I don't know who %name% is but they sound like a good person.",
+        "You were the chosen one! It was said you would destroy TPH, not join them! You were my sibling, %name%, I loved you!",
         "Halt, %name%! The way is shut. It was made by those who are mods, and the mods keep it, until the time comes.",
         "Welcome, %name%! May your beer be laid under an enchantment of surpassing excellence for seven years!",
         "It is perilous to study too deeply the arts of %name%, for good or for ill.",
         "Old %name%, water-lilies bringing, comes hopping to TPH again. Can you hear them singing?",
         "Old %name% is a merry fellow, bright blue their jacket is, and their boots are yellow!",
-        "Give %name% a fire and he's warm for a day. But set fire to him and he's warm for the rest of his life!",
+        "Give %name% a fire and they're warm for a day. But set fire to them and they're warm for the rest of their life!",
```

Comment:
Change `them` to `'em` and I think this will scan better. More slangy but works well.